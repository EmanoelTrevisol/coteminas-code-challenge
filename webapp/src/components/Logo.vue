<template>
  <h1 class="logo">
    mmartan
  </h1>
</template>

<script>
export default {};
</script>

<style lang="stylus" scoped>
h1 {
	margin: 0 10px 0;
	color: $brand-logo;
	font-weight: 500;
	letter-spacing: 1.25px;
	width: 150px;
}
</style>
