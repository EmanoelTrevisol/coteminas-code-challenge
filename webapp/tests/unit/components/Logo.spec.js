import { shallowMount } from '@vue/test-utils';
import Logo from '@/components/Logo';

describe('Logo.vue', () => {
  it('renders Logo component', () => {
    const wrapper = shallowMount(Logo);

    expect(wrapper.html()).toMatchSnapshot();
  });
});
