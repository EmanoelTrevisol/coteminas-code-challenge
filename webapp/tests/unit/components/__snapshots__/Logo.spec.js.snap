// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo.vue renders Logo component 1`] = `
<h1 class="logo">
  mmartan
</h1>
`;
